lighthouse-core/scripts/generate-timing-trace.js

```diff
@@ -45,7 +48,8 @@ function saveTraceFromCLI() {
   const lhrObject = JSON.parse(fs.readFileSync(filename, 'utf8'));
   const jsonStr = createTraceString(lhrObject);
 
-  const traceFilePath = `${filename}.timing.trace.json`;
+  const pathObj = path.parse(filename);
+  const traceFilePath = path.join(path.dirname(filename), `${pathObj.name}.timing.trace.json`);
```

Comment:
drive by changes (after noticing it didn't handle `lhr.report.json` as expected)